shlr/java/class.c

```diff
@@ -8376,42 +8388,48 @@ R_API char *r_bin_java_resolve_b64_encode(RBinJavaObj *BIN_OBJ, ut16 idx) {
 		}
 	} else if (strcmp (cp_name, "Long") == 0) {
 		str = malloc (34);
-		out = malloc (34);
-		memset (out, 0, 34);
 		if (str) {
 			snprintf (str, 34, "0x%"PFMT64x, r_bin_java_raw_to_long (item->info.cp_long.bytes.raw, 0));
-			r_base64_encode (out, (const ut8 *) str, strlen (str));
+			out = malloc (34);
+			if (out) {
+				memset (out, 0, 34);
+				r_base64_encode (out, (const ut8 *) str, strlen (str));
```

Comment:
use r_base64_encode_dyn() which does the malloc internally. the memset is unnecessary

Reply:
(because it can be done with just a calloc). but use the api

Reply:
done